oracle.py

```diff
@@ -362,10 +362,6 @@ def get_score(
         #     motif_distance = binaryDistance(np.concatenate((padded_motif,padded_strings),axis=0), pairwise=True)[0,1:] # the first element is the motif we are looking for - take everything after this
         #     dict_return.update({"motif": motif_distance - 1}) # result is normed on 0-1, so dist-1 gives scaling from 0(bad) to -1(good)
 
-        # if energy_weighting:
-        #     for key in dict_return.keys():
-        #         if key != 'energy':
-        #             dict_return[key] = dict_return[key] * np.tanh(np.abs(energies)/2) # positive tahn of the energies, scaled
```

Comment:
Not critical, but what was the reason to remove this part?

Reply:
This code should not have been removed, because pins and pairs and motifs are relevant.

Reply:
So far, to add a new oracle in oracle.py, we just have to create a new child class of OracleBase (mother class of all oracles), and add it in the wrapper class that calls the relevant oracle Oracle.
Right now, OracleNupack just outputs the energies given by Nupack (in the get_score method). We did not include the other scores given by Nupack yet, but we will.
We just have to discuss if we add these other oracle functions of Nupack (pins, pairs) in other classes, or all in the same big OracleNupack. The better would be to stick to previous code and have a very general Nupack oracle that computes all Nupack statistics (pins, pairs), we will have to add a specific parameter in the config file so that the user specifies which Nupack stats he wants if he selects oracle.main = "nupack".
To be discussed !